src/plugin.py

```diff
@@ -226,6 +262,26 @@ def _force_utc(self):
             logging.exception("Can not parse backend response")
             raise UnknownBackendResponse()
 
+        try:
+            steamFolder = get_configuration_folder()
+            accountId = str( int(self._steam_id) - 76561197960265728  )
+            vdfFile = os.path.join(steamFolder, "userdata", accountId, "config", "localconfig.vdf")
+            logging.debug(vdfFile)
```

Comment:
`logging.debug(f"Users Localconfig.vdf {vdfFile}")`
or anything else which would help looking for this in the logs